keyvaluestore/nodereader.go

```diff
@@ -17,5 +19,55 @@ func (reader *NodeReaderImpl) ReadNode(nodeId uint64) (*node, error) {
 		return nil, nil
 	}
 
-	return nil, nil
+	pageSize := os.Getpagesize()
```

Comment:
This will cause trouble if we move the store from one os to the other. We should probably encode the used page size into the file upon initialization of the kv-store.

Reply:
You are right. Moving a store file to a system with a smaller page-size will always result in very sub-optimal performance. We would probably have to rebuild the whole tree if we see a different page-size. 
I think we can ignore this...